specs/wp-invite-users-spec.js

```diff
@@ -94,11 +95,24 @@ testDescribe( `[${ host }] Invites:  (${ screenSize })`, function() {
 			} );
 
 			test.it( 'Sends an invite', function() {
-				return this.invitePeoplePage.inviteSent().then( sent => {
-					assert.equal( sent, true, 'The sent confirmation message was not displayed' );
+				 this.invitePeoplePage.inviteSent().then( sent => {
+				 return assert.equal( sent, true, 'The sent confirmation message was not displayed' );
 				} );
 			} );
 
+			test.it(
+				'Can see pending invite',
```

Comment:
When I test this in a mobile browser size, this step is failing with this error:

`TimeoutError: Timed out waiting for element with css selector of '.sites-navigation [data-tip-target="people"] a' to be clickable`

Are you also seeing that or is it working for you in all browser sizes?